demo/server/config-dev.json

```diff
@@ -19,5 +19,9 @@
   },
   "redis": {
     "url": "redis://127.0.0.1:6379"
+  },
+  "jwtSecret": "raincatcher",
+  "clientUrl": {
+    "portal": "http://localhost:9003/?url=http://localhost:8001"
```

Comment:
[Bug] 💯  That's not goingto work for any other environment apart from local development. 
Having it this way will also mean additional setup instruction to this. 
Overall -1 for this parameter, but we probably need to talk about it to get proper intentions here.